src/Raven.Server/Documents/TransactionOperationsMerger.cs

```diff
@@ -416,6 +630,8 @@ private void NotifyTransactionFailureAndRerunIndependently(List<MergedTransactio
                     try
                     {
                         previous.InnerTransaction.LowLevelTransaction.RetrieveCommitStats(out commitStats);
+
+                        _recordingStatus?.Record(TxInstruction.FinishAsyncAndStartNew);
```

Comment:
Let's use `BeginAsyncCommitAndStartNewTransaction` instead of `FinishAsyncAndStartNew`

Reply:
done